extensions/EXT_texture_norm16/extension.xml

```diff
@@ -66,6 +72,19 @@ interface EXT_texture_norm16 {
 };
   </idl>
 
+  <newfun>
+    <function name="getSupportedInputs" type="sequence&lt;DOMString&gt;">
+      Returns the names of the input types supported by the implementation. As of this writing,
+      possibly valid return values will include "ArrayBufferView", "ImageBitmap", "ImageData",
```

Comment:
Please rephrase: "valid return values are".

Also, could you please define an enumeration for this return value?
https://heycam.github.io/webidl/#idl-enums

Please use lowercase for the enum values. I think not having hyphens is easier to read: e.g. "arraybufferview".

Reply:
Like what we have in the latest?